source/includes/harvest/_applications.md

```diff
@@ -33,6 +33,18 @@ Applications associate [candidates](#candidates) with [jobs](#jobs). There are 2
   "custom_fields": {
     "bio": null,
     "birthday": null
+  },
+  "keyed_custom_fields": {
+    "date_of_birth": {
+      "name": "Birthday",
+      "type": "date",
+      "value": null
```

Comment:
We should probably include an example of date of birth here since dates are always annoying to work with in API responses.